README.md

```diff
@@ -1 +1,8 @@
-# terraform-cdp-lambda
\ No newline at end of file
+# terraform-cdp-lambda
+
+A terraform module for allocating resources for and deploying AWS Lambda functions.
+The module builds a zip for the lambda deployment using the archive provider.
+
+Things the module currently does not do (often suprisingly hard without null resources or similar "hacks"):
```

Comment:
Typo: 'surprisingly'

Forslag:
The module currently does not (often surprisingly hard without null resources or similar "hacks"):
- Automatically deploy layers.
- Deploy an arbitrary set of source files (though it _does_ support including an optional `upsert.sql`).

Reply:
Ah, bra du så flere ting der 😅, jeg fikser!

btw, anbefaler denne funksjonen på GitHub for tilfeller som dette 😜 (trykk og dra blått plusstegn i margen for å velge flere linjer)
![image](https://github.com/nsbno/terraform-cdp-lambda/assets/17160920/aa56eec3-596e-410d-ae7c-1361d29bc938)